components/movementsensor/gpsrtkserial/gpsrtkserial.go

```diff
@@ -352,7 +352,7 @@ func (g *rtkSerial) connectAndParseSourceTable() error {
 	g.logger.Debugf("sourceTable is: %v\n", srcTable)
 
 	g.logger.Debug("got sourcetable, parsing it...")
-	g.isVirtualBase, err = gpsutils.FindLineWithMountPoint(srcTable, g.ntripClient.MountPoint)
+	g.isVirtualBase, err = gpsutils.IsVRS(srcTable, g.ntripClient.MountPoint)
```

Comment:
Open to disagreement on the name for this function. FindLineWithMountPoint didn't make sense to me.

Reply:
how about 
"HasVRSStream" or "CheckVRSStream"

You are returning a bool four whether the source table contains specific values. If this were a function on (s sourcetable) the naming makes absolute sense, but it's a helper that takes in a sourcetabel to check if we're getting a VRS sourcetable.

Reply:
I like HasVRSStream! I'll update it